wsagent/che-core-api-project/src/test/java/org/eclipse/che/api/watcher/server/impl/FileWatcherManagerTest.java

```diff
@@ -84,4 +86,11 @@ public void shouldUnWatchByMatcher() throws Exception {
 
     verify(fileWatcherByPathMatcher).unwatch(ID);
   }
+
+  private static class DummyRootProvider extends RootDirPathProvider {
```

Comment:
Here will be better use Mock instead of Dummy implementation, this is allow avoid duplication

Reply:
not sure it simplifies a lot but fixed, thanks )